lib/media/track/remotevideotrack.js

```diff
@@ -247,19 +262,24 @@ function setupDocumentVisibilityTurnOff(removeVideoTrack) {
  * updates render hints
  */
 function updateRenderHints(removeVideoTrack) {
-  if (!removeVideoTrack._idleTrackSwitchOff) {
+  if (!removeVideoTrack._idleTrackSwitchOff && !removeVideoTrack._renderHints) {
```

Comment:
Why is this check required if you are using `NullObserver`?

Reply:
With NullObserver the observer callbacks are covered. but this  check is required because we call also this function outside the observe callbacks.  For example this function is called from RemoteVideoTrack.start and RemoteVideoTrack.detach.